scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -0,0 +1,79 @@
+/**
+* @ngdoc controller
+* @module superdesk.apps.web_publisher
+* @name WebPublisherManagerController
+* @requires publisher 
+* @requires modal
+* @requires https://docs.angularjs.org/api/ng/type/$rootScope.Scope $scope
+* @description WebPublisherManagerController holds a set of functions used for web publisher
+*/
+WebPublisherManagerController.$inject = ['$scope', 'publisher', 'modal'];
+export function WebPublisherManagerController($scope, publisher, modal) {
+    var self = this;
+    var siteCode;
+
+    self.TEMPLATES_DIR = 'scripts/apps/web-publisher/views';
+
+    /**
+    * @ngdoc method
+    * @name WebPublisherManagerController#changeTab
+    * @param {String} newTabName - name of the new active tab
+    * @description Sets the active tab name to the given value
+    */
+    self.changeTab = function (newTabName) {
+        self.activeTab = newTabName;
+    };
+
+    /**
+    * @ngdoc method
+    * @name WebPublisherManagerController#toogleCreateSite
+    * @description Opens modal window for creating new site
+    */
+    self.toogleCreateSite = () => {
+        siteCode = '';
+        $scope.new = {};
+        self.openSiteModal = !self.openSiteModal;
+    };
+
+    /**
+    * @ngdoc method
+    * @name WebPublisherManagerController#editSite
+    * @param {Object} site - site which is edited
+    * @description Opens modal window for editing site
+    */
+    self.editSite = (site) => {
+        siteCode = site.code;
+        $scope.new = _.pick(site, 'name', 'subdomain');
+        self.openSiteModal = !self.openSiteModal;
+        publisher.setTenant(site.subdomain)
+            .queryMenus().then(menus => $scope.menus = menus);
+    };
+
+    /**
+    * @ngdoc method
+    * @name WebPublisherManagerController#save
+    * @description Saving site
+    */
+    self.save = () => {
+        publisher.manageSite({tenant: $scope.new}, siteCode).then(refreshSites);
```

Comment:
I think it would make sense to return this value, so that people can do stuff like `save.then(...`.... And so you can know when save completed.

In which case, also `@returns {Promise}`